elide-integration-tests/src/test/java/example/Parent.java

```diff
@@ -18,12 +18,7 @@
 import com.yahoo.elide.security.checks.prefab.Role;
 import lombok.ToString;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.JoinColumn;
-import javax.persistence.JoinTable;
-import javax.persistence.ManyToMany;
+import javax.persistence.*;
```

Comment:
You can actually refer to use [our IntelliJ code style template](https://github.com/yahoo/elide/blob/master/CONTRIBUTING.md)